docs/installation/presto-port-configuration.rst

```diff
@@ -4,7 +4,7 @@
 Presto Port Configuration
 =========================
 
-By default, Presto configuration uses 8080 for the HTTP port. If the port is already used on a given node on your cluster, that server will fail to start.
+By default, Presto configuration uses 8080 for the HTTP port. If the port is already in use on a given node on your cluster, that server will fail to start.
```

Comment:
Regarding line 11: I am dismayed and annoyed to learn that netstat -l is not even the tiniest bit portable. scowl.